.all-contributorsrc

```diff
@@ -1835,6 +1835,15 @@
       "contributions": [
         "doc"
       ]
+    },
+    {
+      "login": "marcioggs",
+      "name": "Márcio Gabriel",
+      "avatar_url": "https://avatars.githubusercontent.com/u/2506127?v=4",
+      "profile": "https://github.com/marcioggs",
+      "contributions": [
+        "doc"
+      ]
```

Comment:
It just caught my attention that we're missing a contributors entry in the readme.
We'll fix that by using the bot.

```suggestion
```